keyboards/durgod/k320/board.h

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2020 Don Kjer
```

Comment:
Is it worth updating these to `Copyright 2020, 2021 ...`?

Reply:
Agreed